packages/ppx/test/native/At_rule_test.re

```diff
@@ -67,6 +67,13 @@ let media_query_tests = [
       CssJs.style([|CssJs.media({js|screen and (color) |js}, [||])|])
     ],
   ),
+  (
+    "(screen and (color))",
+    [%expr [%cx "@media (screen and (color)) {}"]],
+    [%expr
+      CssJs.style([|CssJs.media({js|(screen and (color))|js}, [||])|])
+    ],
+  ),
```

Comment:
This test fails with `"[%ocaml.error \"Parse error while reading token 'and'\"]`, probably because of the parsing rules, but I am not sure where exactly. Any pointers @davesnx ?

Reply:
Parse error comes from the properties parser, so eventually it would need to understand and as well. There're all media-queries combinations on the properties parser under Parser.re

Reply:
All media_ prefixed from Parser.re are references to media-querie parsing: https://github.com//davesnx/styled-ppx/blob/28035df2f39cd5337f7de5098985f96f769cbf3f/packages/reason-css-parser/lib/Parser.re#L479-L480

Reply:
BTW, do we need to keep the above lines or weed them out in this PR?